develop-docs/self-hosted/index.mdx

```diff
@@ -108,6 +108,37 @@ By default Sentry sends anonymous usage statistics to the Sentry team. It helps
 
 You can disable this feature by adding `SENTRY_BEACON = False` into `sentry.conf.py` file.
 
+## Errors-only Self-hosted Sentry
+
+**Disclaimer: This is an experimental beta feature. This means that features and workflows are not completely tested, so use at your own risk!**
+
```

Comment:
```suggestion
<Note>
This is an experimental beta feature. This means that features and workflows are not completely tested, so use at your own risk!
</Note>

```
Might be nice to make this a note the way it would be in user-facing docs :)